src/Symfony/Component/Workflow/Registry.php

```diff
@@ -63,6 +63,13 @@ public function get($subject, $workflowName = null)
         return $matched;
     }
 
+    /**
+     * @param SupportStrategyInterface $supportStrategy
```

Comment:
see #33198

Reply:
The doc block is correct here. The interface was replaced by a new one in 4.1.